frontend/src/metabase/browse/components/MetricsTable.tsx

```diff
@@ -0,0 +1,292 @@
+        <TableColumn {...collectionProps} width={`${collectionWidth}%`} />
+
+        {/* <col> for Description column */}
+        <TableColumn {...descriptionProps} width={`${descriptionWidth}%`} />
+
+        <Columns.RightEdge.Col />
+      </colgroup>
+      <thead>
+        <tr>
+          <SortableColumnHeader
+            name="name"
+            sortingOptions={sortingOptions}
+            onSortingOptionsChange={handleSortingOptionsChange}
+            {...nameProps}
+            style={{ paddingInlineStart: ".625rem" }}
+            columnHeaderProps={{
+              style: { paddingInlineEnd: ".5rem" },
+            }}
+          >
+            {t`Name`}
+          </SortableColumnHeader>
+          <SortableColumnHeader
+            name="collection"
+            sortingOptions={sortingOptions}
+            onSortingOptionsChange={handleSortingOptionsChange}
+            {...collectionProps}
+            columnHeaderProps={{
+              style: {
+                paddingInline: ".5rem",
+              },
+            }}
+          >
+            {t`Collection`}
+          </SortableColumnHeader>
+          <SortableColumnHeader
+            name="description"
+            sortingOptions={sortingOptions}
+            onSortingOptionsChange={handleSortingOptionsChange}
+            {...descriptionProps}
+            columnHeaderProps={{
+              style: {
+                paddingInline: ".5rem",
+              },
+            }}
+          >
+            {t`Description`}
+          </SortableColumnHeader>
+          <Columns.RightEdge.Header />
+        </tr>
+      </thead>
+      <TBody>
+        {skeleton ? (
+          <Repeat times={7}>
+            <MetricRow />
+          </Repeat>
+        ) : (
+          sortedMetrics.map((metric: MetricResult) => (
+            <MetricRow metric={metric} key={metric.id} />
+          ))
+        )}
+      </TBody>
+    </Table>
+  );
+}
+
+function MetricRow({ metric }: { metric?: MetricResult }) {
+  const dispatch = useDispatch();
+
+  const handleClick = useCallback(
+    (event: MouseEvent) => {
+      if (!metric) {
+        return;
+      }
+
+      // do not trigger click when selecting text
+      const selection = document.getSelection();
+      if (selection?.type === "Range") {
+        event.stopPropagation();
+        return;
+      }
+
+      const { id, name } = metric;
+      const url = Urls.metric({ id, name });
+      const subpathSafeUrl = Urls.getSubpathSafeUrl(url);
+
+      // TODO:
+      // trackMetricClick(metric.id);
+
+      event.preventDefault();
+      event.stopPropagation();
+
+      if ((event.ctrlKey || event.metaKey) && event.button === 0) {
+        Urls.openInNewTab(subpathSafeUrl);
+      } else {
+        dispatch(push(url));
+      }
+    },
+    [metric, dispatch],
+  );
+
+  return (
+    <TableRow onClick={handleClick} tabIndex={0}>
+      <NameCell metric={metric} />
+      <CollectionCell metric={metric} />
```

Comment:
In models table the collection name changes color on hover.
:x:  In metrics table it does not.

Reply:
I've updated the code to share this component.